spec/spec_helper.rb

```diff
@@ -0,0 +1,6 @@
+require "logstash/devutils/rspec/spec_helper"
+require 'logstash/devutils/rspec/logstash_helpers'
+
+require "logstash/filters/base"
+require "logstash/outputs/base"
+require "logstash/namespace"
```

Comment:
for now  this -> https://github.com/elasticsearch/logstash-devutils/blob/master/lib/logstash/devutils/rspec/logstash_helpers.rb#L2 adds the three of them, so good to be removed.